nexteuropa_newsroom.ds.inc

```diff
@@ -193,22 +173,14 @@ function nexteuropa_newsroom_ds_field_settings_info() {
     'post_date' => array(
       'weight' => '2',
       'label' => 'hidden',
-      'format' => 'ds_post_date_dsm_long_type_without_time',
+      'format' => 'ds_post_date_newsroom_date_only',
       'formatter_settings' => array(
         'ft' => array(
           'func' => 'theme_ds_field_nexteuropa_formatters_meta_item',
```

Comment:
Dependency should be declared on nexteuropa_formatters for this to work.
README should be also updated on this dependency.